src/NPacMan.Game/Ghost.cs

```diff
@@ -2,30 +2,35 @@
 {
     public class Ghost
     {
-        private readonly IGhostStrategy _homeStrategy;
         public string Name { get; }
         public int X { get; }
         public int Y { get; }
         public IGhostStrategy Strategy { get; }
+        public IGhostStrategy CurrentStrategy { get; }
+        private IGhostStrategy _homeStrategy { get; }
 
-        public Ghost(string name, int x, int y, IGhostStrategy strategy, IGhostStrategy homeStrategy)
+        public int HomeLocationX => ((MoveHomeGhostStrategy)_homeStrategy).X;
```

Comment:
This code only exists as an inspection point for tests

Reply:
Maybe the Home Location should just be set on the Ghost? My thoughts are is that we pass in ghosts to the `IGhostStrategy` on the `Move(Ghost, Game)`

Reply:
Yeah,  I was thinking something similar.  Our current design might be over complicating things as well :-)

Reply:
Yeah, maybe a little over-engineered... I think it didn't help that we didn't fully understand the requirements for the ghosts originally.